public/bitemp_graph.js

```diff
@@ -583,7 +585,19 @@ var barChart = function() {
         var date = new Date(input).toISOString();
```

Comment:
Maybe add a comment for some of these conditionals because reading it isn't necessarily easy